sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2708,6 +2708,7 @@ export interface ServiceGetAccountInfoHeaders {
     date?: Date;
     // (undocumented)
     errorCode?: string;
+    isHnsEnabled?: boolean;
```

Comment:
Does other languages use similar name? I wonder whether we want a more user friendly full name instead of `Hns`.

Reply:
The change originated from us. I made the swagger change. https://github.com/Azure/azure-rest-api-specs/pull/11871
I will ask if they would prefer a better name.

Reply:
Renamed.